src/modules/rover_pos_control/RoverPositionControl.hpp

```diff
@@ -155,6 +158,11 @@ class RoverPositionControl final : public ModuleBase<RoverPositionControl>, publ
 	/* previous waypoint */
 	matrix::Vector2d _prev_wp{0, 0};
 
+	enum class VelocityFrame {
+		NED,
+		BODY,
+	} _velocity_frame{VelocityFrame::NED};
```

Comment:
Why are the frame specific for the rover pos control?

Reply:
It's not handled yet, but I'm assuming we're going to add it. Right now ORB_ID(trajectory_setpoint) is always NED, with things like body frame velocity setpoints being converted by the mavlink receiver. I'd like to push all of that into the controllers and keep mavlink as a dumb passthrough.